specification/compute/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/examples/ContainerGroupsListAll.json

```diff
@@ -0,0 +1,60 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "api-version": "2017-08-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/resourceGroups/demo/providers/Microsoft.ContainerInstance/containerGroups/demo1",
+            "location": "west us",
+            "name": "demo1",
+            "properties": {
+              "containers": [
+                {
+                  "name": "demo1",
+                  "properties": {
+                    "command": null,
+                    "environmentVariables": [
+                      {
+                        "name": "a",
+                        "value": "ab"
+                      }
+                    ],
+                    "image": "nginx",
+                    "instanceView": null,
+                    "ports": [
+                      {
+                        "port": 80
+                      }
+                    ],
+                    "resources": null
+                  }
+                }
+              ],
+              "imageRegistryCredentials": null,
+              "ipAddress": {
+                "ip": "10.0.0.0",
```

Comment:
you marked `type` as required, but haven't provided it here. Check whether the Swagger is correct or this example and fix either accordingly